rundoozer/rundoozer

```diff
@@ -214,10 +217,21 @@ def main():
 
     doozer_args = clean_args(doozer_args)
 
-    cfg = dotconfig.Config('rundoozer', 'settings',
-                           template=CLI_CONFIG_TEMPLATE,
-                           envvars=CLI_ENV_VARS,
-                           cli_args=args)
+    settings_file = None
+    if 'DOOZER_SETTINGS' in os.environ:
+        settings_file = os.environ['DOOZER_SETTINGS']
+
+    try:
+        cfg = dotconfig.Config('rundoozer', 'settings',
+                               template=CLI_CONFIG_TEMPLATE,
+                               envvars=CLI_ENV_VARS,
+                               cli_args=args,
+                               file_override=settings_file)
+    except Exception as e:
+        raise
```

Comment:
You raise, this block will exit before getting to any print part, won't it?

Reply:
Yep... was debug. Removed.